frontend/src/metabase/visualizations/click-actions/actions/NativeQueryClickFallback.tsx

```diff
@@ -25,16 +25,28 @@ export const NativeQueryClickFallback: LegacyDrill = ({ question }) => {
       section: "info",
       type: "custom",
       view: ({ dispatch }) => (
-        <Text>{jt`${(
-          <Button
-            key="button"
-            variant="subtle"
-            p={0}
-            onClick={() => dispatch(setUIControls({ modal: MODAL_TYPES.SAVE }))}
-          >
-            {t`Save`}
-          </Button>
-        )} this question to drill-thru.`}</Text>
+        <Text
+          styles={{
+            root: {
+              display: "flex",
+              alignItems: "baseline",
+              gap: "0.25rem",
+            },
+          }}
+        >
+          {jt`${(
+            <Button
+              key=""
+              variant="subtle"
+              p={0}
+              onClick={() =>
+                dispatch(setUIControls({ modal: MODAL_TYPES.SAVE }))
+              }
+            >
+              {t`Save`}
+            </Button>
+          )} this question to drill-thru.`}
```

Comment:
Minor point, but should this be `through`? I think we usually use `drill-through` in the product and `drill-thru` in the code but I might be mistaken

Reply:
Yes!

Reply:
@noahmoss please update if you can